packages/contracts-bedrock/contracts/libraries/trie/MerkleTrie.sol

```diff
@@ -160,7 +160,7 @@ library MerkleTrie {
             } else {
                 // Nodes smaller than 31 bytes aren't hashed.
                 require(
-                    BytesUtils.toBytes32(currentNode.encoded) == currentNodeID,
+                    bytes32(currentNode.encoded) == currentNodeID,
```

Comment:
This was suggested by Spearbit. Has an identical effect, and is available since 0.8.5.